src/app/routes/utils/sharedDataTransformers/transformCPSAssetToIndexPage/index.ts

```diff
@@ -0,0 +1,151 @@
+        locators?: { assetUri: string };
+        indexImage: { path: string; altText: string };
+        media?: {
+          format: string;
+          versions: {
+            durationISO8601: string;
+          }[];
+        };
+        summary: string;
+        timestamp: number;
+      }[];
+      strapline?: {
+        name: string;
+      };
+    }[];
+  };
+};
+
+type CpsPageData = {
+  article: CpsArticle;
+  secondaryData: { mostRead?: MostReadData };
+};
+
+export default (cpsAsset: CpsPageData) => {
+  const {
+    article: {
+      metadata: {
+        analyticsLabels: {
+          contentId,
+          counterName: pageIdentifier,
+          cps_asset_type: contentType,
+        },
+        title,
+        summary: description,
+        type: pageType,
+      },
+      content: { groups },
+    },
+    secondaryData: { mostRead },
+  } = cpsAsset;
+
+  const curations: Curation[] = groups.map(
+    ({ type, items, strapline }, curationIndex) => {
+      const summaries: Summary[] = items.map(
+        ({
+          id,
+          name,
+          headlines,
+          uri,
+          locators,
+          indexImage,
+          media,
+          summary,
+          timestamp,
+        }) => {
+          const duration = media?.versions?.[0].durationISO8601;
+
+          return {
+            title: name || headlines?.headline || '',
+            type: media?.format || 'article',
+            duration,
+            lastPublished: new Date(timestamp).toISOString(),
+            imageUrl: `https://ichef.bbci.co.uk/ace/ws/{width}${indexImage?.path}.webp`,
+            imageAlt: indexImage?.altText || '',
+            id,
+            link: uri || `https://www.bbc.com${locators?.assetUri}`,
+            description: summary,
+            isLive: false,
+          };
+        },
+      );
+
+      const curationTitle = strapline?.name || '';
+
+      /**
+       * Display each curation as a hierarchical collection,
+       * but if there are less than 3 promos,
+       * display the curation a collection
+       * */
+      const visualProminence: VisualProminence =
+        summaries.length < 3 ? NORMAL : HIGH;
+
+      return {
+        curationType: 'vivo-stream',
+        curationId: type,
+        title: curationIndex > 0 ? curationTitle : '',
+        position: curationIndex + 1,
+        visualProminence,
+        visualStyle: 'NONE',
+        summaries,
+      };
+    },
+  );
+
+  if (mostRead) {
+    curations.push({
+      curationId: 'urn:bbc:onward-journeys:most:read?=site=News',
+      curationType: 'most-popular',
+      position: curations.length,
+      title: 'Most Read',
+      visualProminence: 'NORMAL',
+      visualStyle: 'RANKED',
+      mostRead,
+    });
+  }
+
+  return {
+    title,
+    description,
+    curations,
+    metadata: {
+      type: pageType,
+      atiAnalytics: {
+        contentId,
+        contentType: contentType?.toUpperCase(),
+        pageIdentifier,
+        pageTitle: title,
+      },
```

Comment:
The ATI analytics are going to have FIX data instead of Topic page data.. is this OK?